src/npc.h

```diff
@@ -554,7 +554,7 @@ class npc : public player
  ~npc() override;
 
     /** Construct an NPC with standard stats */
-    static npc standard();
+    static npc make_standard();
```

Comment:
> Would need to override each constructor separately?

Not for now. The compiler will make a set of default constructors that will call associated constructors of the base class implicitly. There are no custom constructors in `npc`.